app/Jobs/SendNewRecordingNotifications.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Friend;
+use App\Jobs\Job;
+use App\Phone\Exceptions\BlacklistedPhoneNumberException;
+use App\Phone\TwilioClient;
+use App\PhoneNumber;
+use App\Recording;
+use App\User;
+use Illuminate\Log\Writer as Logger;
+
+class SendNewRecordingNotifications extends Job
```

Comment:
Yah I think whether it's there or somewhere else, I'd prefer we extract these two back out--even if it means we have to cover the exception in multiple places. Thanks!